config/app_local.default.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/*
+ * Local configuration file to provide any overrides to your app.php configuration.
+ */
+return [
+    /**
```

Comment:
We shouldnt use docblocks when those are comment blocks.
Docblocks are specificially for variables, methods and alike and require @ tags inside.

That's a misconcept used already in core/app. This can also be done separately though as follow up.